mlos_bench/mlos_bench/tests/services/remote/azure/conftest.py

```diff
@@ -24,7 +25,18 @@ def config_persistence_service() -> ConfigPersistenceService:
 
 
 @pytest.fixture
-def azure_vm_service(config_persistence_service: ConfigPersistenceService) -> AzureVMService:
+def azure_auth_service(config_persistence_service: ConfigPersistenceService,
+                       monkeypatch: pytest.MonkeyPatch) -> AzureAuthService:
+    """
+    Creates a dummy AzureAuthService for tests that require it.
+    """
+    auth = AzureAuthService(config={}, parent=config_persistence_service)
+    monkeypatch.setattr(auth, "get_access_token", lambda: "TEST_TOKEN")
```

Comment:
Seems ok.  But doesn't do any testing for the logic for out of date tokens, right?

Reply:
no, we don't have tests for that yet. I'll add them when we switch to azure-identity - then we'll have an API that we can mock/monkey patch in our unit tests

Reply:
Yeah, that makes sense to me too.  I made an issue on that #402